src/panels/logbook/ha-logbook.ts

```diff
@@ -25,20 +25,17 @@ class HaLogbook extends LitElement {
   // @ts-ignore
   private _rtl = false;
 
-  protected updated(changedProps: PropertyValues) {
-    super.updated(changedProps);
-    if (!changedProps.has("hass")) {
-      return;
-    }
+  protected shouldUpdate(changedProps: PropertyValues) {
     const oldHass = changedProps.get("hass") as HomeAssistant | undefined;
-    if (oldHass && oldHass.language !== this.hass.language) {
-      this._rtl = computeRTL(this.hass);
-    }
+    return (
+      changedProps.has("entries") ||
+      (oldHass && oldHass.language !== this.hass.language)
+    );
   }
 
-  protected firstUpdated(changedProps: PropertyValues) {
-    super.firstUpdated(changedProps);
+  protected performUpdate() {
     this._rtl = computeRTL(this.hass);
```

Comment:
You are changing `_rtl` now on every prop update (a lot), should only be done when the language is changed (that is almost never)

Reply:
You're right, I was thinking it's called only when `shouldUpdate` returns `true`.